.circleci/config.yml

```diff
@@ -14,16 +14,18 @@ jobs:
 
   build-and-test:
     macos:
-      xcode: 11.6.0 # Specify the Xcode version to use
+      xcode: 12.0.1 # Specify the Xcode version to use
 
     steps:
       - checkout
+      
       - run:
-          name: Install realm/SwiftLint
-          command: brew update-reset && brew install swiftlint
+          name: Pod install
+          command: pod install
+
       - run: 
           name: Swift Linting
-          command: swiftlint lint
+          command: ./Pods/SwiftLint/swiftlint lint
```

Comment:
Can we add these to our MakeFile and call the targets? It will be easier to run these commands locally.